### ISSUE TITLE
[SCREENREADER]: Facility Locator Search for field should announce itself as a combobox instead of button meu

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#711]
## Issue
The Search For field announces itself as a button menu, which isn't the clearest for assistive devices. It's based on Downshift, so hoping we can modify it to announce as a combobox. I'm linking to the WAI-ARIA guidance for a combobox without an autocomplete, which looks like what we want here.

## Audit Finding
* Note 9, Defect 2

## Acceptance Criteria
* As a screen reader user, I want the Search for dropdown (combobox) to announce itself as a "combobox" and retain all existing functionality.

## Environment
* Windows 10
* IE11 + JAWS
* https://staging.va.gov/find-locations/ (Searched for "Austin, TX" and "Salt Lake City"

## Possible Fixes (optional)
I'll defer offering a refactor fix, but instead point to the WAI-ARIA pattern: https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.0pattern/combobox-autocomplete-none.html#ex_label

## WCAG or Vendor Guidance (optional)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs
![Screen Shot 2019-07-10 at 6.45.27 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/6e0d76e0-4a9d-4d54-ae43-899739d4fc5b)

**Comment by 1Copenut** [comment]
Closing this as not a defect per guidance from VA 508 office.

/cc @Karak888
